service/vc/dbtest/container.go

```diff
@@ -317,6 +345,30 @@ func (dc *DatabaseContainer) ContainerID() string {
 	return dc.containerID
 }
 
+// ReadPasswordFromContainer extracts the randomly generated password from a file inside the container.
+// This is required because YugabyteDB, when running in secure mode, doesn't allow default passwords
+// and instead generates a random one at startup.
+// If no password is found, the default one will be returned.
+func (dc *DatabaseContainer) ReadPasswordFromContainer(t *testing.T, filePath string) string {
+	t.Helper()
+	output := dc.ExecuteCommand(t, []string{"cat", filePath})
+
+	scanner := bufio.NewScanner(strings.NewReader(output))
+	re := regexp.MustCompile(`(?i)^password:\s*(.+)$`)
```

Comment:
major: There is no reason to read the input line by line. You should make the regular expression multi-line.
minor: This should be pre-compiled outside the method.
minor: use meaningful name.
```suggestion
	passwordRegexp := regexp.MustCompile(`(?i)(?m)^password:\s*(.+)$`)
```